src/cli/annotations.rs

```diff
@@ -31,8 +31,13 @@ impl Execute for Stats {
     where
         W: WriteColor,
     {
+<<<<<<< HEAD
+        let document = Document::load(self.file)
+            .with_context(|| format!("Failed to read PDF from: {:?}", self.file))?;
+=======
         let document = Document::load(&self.file)
             .with_context(|| format!("Failed to read PDF from: {:?}.", self.file))?;
+>>>>>>> 85f3c4e8c9845f468b57c250633363d773b01d2b
```

Comment:
Hi @Brijeshthummar02, you might want to remove this :-)
```suggestion
        let document = Document::load(self.file)
            .with_context(|| format!("Failed to read PDF from: {:?}.", self.file))?;
```